dockers/docker-ptf/Dockerfile.j2

```diff
@@ -100,4 +100,4 @@ COPY ["supervisord.conf", "sshd.conf", "ptf_nn_agent.conf", "/etc/supervisor/con
 
```

Comment:
There is a "main" config file in /etc/supervisor, and that file includes the /etc/supervisor/conf.d/ path.